tests/test_parameters.py

```diff
@@ -0,0 +1,165 @@
+"""Tests of ParameterGenerator() methods involving drawing parameters."""
+import psfws
+import numpy as np
+import pandas as pd
+
+
+def dict_test_helper(thing, p_list, x):
+    """Test whether all keys in p_list exist in p_list and checks lengths."""
```

Comment:
All keys in *thing*?

Reply:
I think that I meant for `x` to always be `h`, so that the length match test is singling out 'h' to compare with all the others (since `h` is in some sense the independent variable) -- but I realize that is not necessary as long as 'h' is in the list I give the test of parameter names to check!